keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -492,6 +483,32 @@ These are the common steps to delete a `Bucket`. Note: there are atypical workfl
 + If the release policy is "Retain" then the `Bucket` remains "Released" and it can potentially be reused.
 + When the COSI controller sees the `Bucket`'s `conditions[x].Reason is "Deleted", it deletes all the finalizers and the real deletions occur.
 
+###  Setting Access Permissions
+#### Dynamic Provisioning
+Incoming BucketAccessRequest either contains a ServiceAccountName where cloud provider supports identity integration or a accessSecretName. In both cases, the incoming BAR representing a request for a user to access the `Bucket`.
+When requesting access to a bucket, workloads will go through the following  scenarios described here:
++  New User and New/Existing Bucket: In this scenario, we do not have user account in the backend storage system as well as no access for this user to the `Bucket`. 
```

Comment:
_New/Existing Bucket_ is not a scenario since all buckets are either new or existing. Is it correct for this scenario to simply be called "New User"?

Reply:
Sure, myself doing a binary table I guess